app/controllers/subscriptions_controller.rb

```diff
@@ -49,9 +51,9 @@ def ordered_people
   end
 
   # Override so we can pass preferred_labels from mailing_list
-  def render_emails(people)
+  def render_emails(people, separator)
```

Comment:
```suggestion
  def render_emails(people, separator = ',')
```
Let's keep backward compatibility, in case some wagon still relies on passing in just one argument here.